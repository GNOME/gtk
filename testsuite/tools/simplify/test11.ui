<interface>
  <object class="GtkPaned">
    <child>
      <object class="GtkLabel">
        <property name="label">start</property>
      </object>
    </child>
    <child>
      <object class="GtkLabel">
        <property name="label">end</property>
      </object>
    </child>
  </object>
</interface>
