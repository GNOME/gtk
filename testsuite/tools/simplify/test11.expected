<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkPaned">
    <property name="start-child">
      <object class="GtkLabel">
        <property name="label">start</property>
      </object>
    </property>
    <property name="end-child">
      <object class="GtkLabel">
        <property name="label">end</property>
      </object>
    </property>
  </object>
</interface>
